src/pages/stablecoins.tsx

```diff
@@ -614,7 +614,7 @@ const StablecoinsPage = ({ markets, marketsHasError }) => {
           <InfoBanner emoji="⚠️" isWarning>
```

Comment:
I would suggest we place this on the InfoBanner as a margin bottom instead:

```suggestion
          <InfoBanner emoji="⚠️" isWarning pb="4">
```

Also @konopkja, does 1rem look like enough here for you? The padding in the info banner and the margin between it and the header is 1.5rem for reference